protocol/communities/community.go

```diff
@@ -857,6 +857,24 @@ func (o *Community) AddCommunityTokensMetadata(token *protobuf.CommunityTokenMet
 	return o.config.CommunityDescription, nil
 }
 
+func containsToken(tokens []*protobuf.CommunityTokenMetadata, symbol string) bool {
```

Comment:
I find it a bit dangerous to use a symbol as a key for community tokens, it can get repetitive. The unique identifier for a community token is contractAddress+chainId

Reply:
Tokens inside community has unique symbols.